pyblazing/pyblazing/apiv2/context.py

```diff
@@ -1158,7 +1200,7 @@ def drop_table(self, table_name):
         """
         self.add_remove_table(table_name, False)
 
-    def _parseSchema(self, input, file_format_hint, kwargs, extra_columns):
+    def _parseSchema(self, input, file_format_hint, kwargs, extra_columns, ignore_missing_paths):
```

Comment:
when and how is ignore_missing_pathjs set

Reply:
See my comment above